projects/examples/src/components/spotlight-search/spotlight-search-example.component.ts

```diff
@@ -35,6 +35,8 @@ export class SpotlightSearchExampleComponent implements OnInit, OnDestroy {
             ['placeholder']: [''],
         });
 
+        this.actionsSearchProvider.register();
```

Comment:
Doesn't really matter because it's an example, but should this be moved to ngOnInit?